go/apps/api/routes/v2_keys_get_key/handler.go

```diff
@@ -80,8 +77,17 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 		)
 	}
 
+	keyData := db.ToKeyData(key)
+	if keyData == nil {
+		return fault.New("failed to process key data",
+			fault.Code(codes.App.Internal.ServiceUnavailable.URN()),
+			fault.Internal("key data processing failed"),
```

Comment:
setting an internal error is redundant here, `fault.New` already uses the first arg to construct an error chain

Reply:
This whole check is redundant I'm pretty sure today, because type system enforces that. But, it would be good to test keyData functions so I'm doing that